Sofa/Component/Constraint/Projective/src/sofa/component/constraint/projective/FixedTranslationConstraint.h

```diff
@@ -91,7 +91,8 @@ class FixedTranslationConstraint : public core::behavior::ProjectiveConstraintSe
 
 protected:
     template <class DataDeriv>
-    void projectResponseT(const core::MechanicalParams* mparams, DataDeriv& dx);
+    void projectResponseT(DataDeriv& dx,
```

Comment:
Is there a reason to pass the `std::function` by copy?

Reply:
dont think so